libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -211,6 +235,8 @@ export interface IExtractorOptions {
   /**
    * The folder where the PNPM "node_modules" folder is located. This is used to resolve packages linked
    * to the PNPM virtual store.
+   * Can't be used with subspaces, if subspaces field is specified, we will use pnpmInstallFolder configured
+   * for each subspace and ignore this.
```

Comment:
Update this to throw if both are provided instead.

Reply:
Fixed in my https://github.com/EscapeB/rushstack/pull/1